apple-pie/index.html

```diff
@@ -4,36 +4,53 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- Remember to link your styles -->
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
-    Apple Pie
 
-    This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
-    be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! 
+    <div class="title">
+      <img class="apple" src="/apple-pie/images/apple-pie.jpg">
```

Comment:
```suggestion
      <img class="apple" src="./images/apple-pie.jpg">
```
Arriba lo tenias perfecto! si le pones /apple-pie le estas diciendo que dentro de la carpeta apple-pie busque una carpeta apple-pie (que no existe porque ya estas en ella jejej)